docs/next/guides/accessibility/keyboard-shortcuts.md

```diff
@@ -0,0 +1,189 @@
+
+Please keep in mind that you can get already registered contexts
+using `getContext` method and even register new context using `addContext`. To switch to another context use `setActiveContextName` method.
+
+### Removing keyboard shortcuts
+To remove an already registered keyboard shortcut (such as one of the default keyboard shortcuts), you need to search for it in the relevant context and refer to it by:
+
+- Either the key variant
+- Or the namespace
+
+Use the context's method `removeShortcutByVariants` to remove all shortcuts registered for given key variants (note that it is possible that there are multiple actions registered for a single keyboard shortcut):
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.removeShortcutByVariants([['enter']]);
+```
+
+Use the context's method `removeShortcutByNamespace` to remove all shortcuts registered in a certain namespace:
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.removeShortcutByNamespace('NAMESPACE_ID');
+```
+
+### Adding custom keyboard shortcuts
+
+Use the context's `addShortcut` method to register an action for a given keyboard shortcut.
+
+Within a single context, there might be multiple actions registered for the same keyboard shortcut. Your action will be simply added at the end of the stack of already defined actions.
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {} });
+```
+
+If your action must run before a certain other action, you can refer to the other action by its namespace:
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {}, position: 'before', relativeToNamespace: 'ANOTHER_NAMESPACE_ID' });
+```
+
+If your action must run only if some specific precondition is met, you can check for the precondition using a function provided to the `runAction` property:
+
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {}, runAction: () => hot.getSelected() !== void 0 });
+```
+
+### Replacing default keyboard shortcuts
+To replace some keyboards shortcut's action by another one action you have to choose proper context, remove already registered shortcut and register another one.
+
+```js
+const gridContext = hot.getShortcutManager().getContext('grid');
+
+gridContext.removeShortcutByVariants(['enter']);
+gridContext.addShortcut({ namespace: 'NAMESPACE_ID', variants: [['enter']], callback: () => {} });
+```
+
+### Using `beforeKeyDown` hook
+Please keep im mind that there is an extra possibility to stop shortcut's action execution. You can return `false` in callback
+to the hook for doing it.
```

Comment:
@wszymanski it is hard to understand the usefulness of this information. Is this used to cancel the execution of all shortcuts or just a single one?

Reply:
Both options are possible. By simply returning `false` all shorcuts are disabled. After checking `event` and their `key` property just single one shortcut can be cancelled.

Please keep in mind that In the callback there can be some conditional, for example related to user application state.

Reply:
Thanks. Can you change the snippet to show the variant that checks `event.key` and cancels just one shortcut? Anyone can deduct from it how to cancel all shortcuts.